packages/runner/lib/app.ts

```diff
@@ -4,12 +4,13 @@ import { stringifyError } from '@nangohq/utils';
 import { logger } from './utils.js';
 import { monitorProviders } from '@nangohq/shared';
 import { register } from './register.js';
+import { envs } from './env.js';
 
 const providersMonitorCleanup = await monitorProviders();
 
 try {
     const port = parseInt(process.argv[2] || '') || 3006;
-    const id = process.argv[3] || process.env['RUNNER_ID'] || 'unknown-id';
```

Comment:
I think we are missing this value for onprem / in the helm chart
see last thread from the onprem customers

Reply:
good point. RUNNER_ID is irrelevant when using a remote runner which is why this value is not set. maybe we could come up with a better id than `unknown-id` but I don't think it is a big deal right now

Reply:
I'm just worried it's going to be undefined, instead of a string, which could break things I guess